src/_components/CentreModal/CentreModal.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createOfficeLocation } from '../../_actions/officeLocations.actions';
+
+import CentreModalComponent from '../../components/CentreModal/CentreModal';
+
+class CentreModal extends Component {
+  state = {
+    name: '',
+    country: ''
+  };
+
+  onAddCentre = (event) => {
+    this.setState({ name: event.target.value });
+  };
+
+  onAddCountry = (event) => {
+    this.setState({ country: event.target.value });
+  };
+
+  handleSubmit = () => {
+    const newCentre = {
+      centre_name: this.state.name,
+      centre_country: this.state.country
+    };
+    this.props.createOfficeLocation(newCentre);
+  };
+  render() {
+    return (
+      <CentreModalComponent
```

Comment:
This component only renders a different component with nothing in between, therefore, combine it with `CentreModal` definition in `/components/*` folder